drivers/shared/executor/procstats/getstats.go

```diff
@@ -74,11 +74,11 @@ func (lps *linuxProcStats) scanPIDs() {
 	}
 }
 
-func (lps *linuxProcStats) cached() ProcUsages {
+func (lps *taskProcStats) cached() ProcUsages {
 	return lps.cache
 }
```

Comment:
this doesn't appear to be used and isn't threadsafe so perhaps remove?

Reply:
Good catch.